app/src/main/java/app/revanced/integrations/patches/PlayerOverlayPatch.java

```diff
@@ -0,0 +1,45 @@
+package app.revanced.integrations.patches;
+
+import android.widget.ImageView;
+
+import app.revanced.integrations.settings.SettingsEnum;
+import app.revanced.integrations.utils.ReVancedUtils;
+
+public class PlayerOverlayPatch {
+    private static final boolean HIDE_PLAYER_OVERLAY = SettingsEnum.HIDE_PLAYER_OVERLAY.getBoolean();
+    private static final int DEFAULT_OPACITY = (int) SettingsEnum.CHANGE_PLAYER_OVERLAY_OPACITY_VALUE.defaultValue;
+    private static int opacity = DEFAULT_OPACITY;
+
+    static {
+        if (!HIDE_PLAYER_OVERLAY)
+            loadCustomOpacityValue();
+    }
+
+    public static void hidePlayerOverlay(ImageView imageView) {
+        if (!HIDE_PLAYER_OVERLAY) return;
+
+        imageView.setImageResource(android.R.color.transparent);
+    }
```

Comment:
Hiding becomes unnecessary if a custom opacity value option exists, as you can set alpha to 0.  Please remove this method.